cogs/strike.py

```diff
@@ -473,56 +483,48 @@ async def _confirm_manual_add_strike(self, strike_user: discord.User | discord.M
                 if _audit_log_entry.target.id == strike_user.id  # NOTE: IDs are checked here rather than the objects themselves as the audit log provides an unusual object type in some cases.
             )
         except (StopIteration, StopAsyncIteration):
-            IRRETRIEVABLE_AUDIT_LOG_MESSAGE: Final[str] = (
-                f"Unable to retrieve audit log entry of {str(action)!r} action "
-                f"on user {str(strike_user)!r}"
-            )
-
             logger.debug("Printing 5 most recent audit logs:")
             debug_audit_log_entry: discord.AuditLogEntry
             async for debug_audit_log_entry in main_guild.audit_logs(limit=5):
                 logger.debug(debug_audit_log_entry)
 
+            IRRETRIEVABLE_AUDIT_LOG_MESSAGE: Final[str] = (
+                f"Unable to retrieve audit log entry of {str(action)!r} action "
+                f"on user {str(strike_user)!r}"
+            )
             raise NoAuditLogsStrikeTrackingError(IRRETRIEVABLE_AUDIT_LOG_MESSAGE) from None
 
         if not audit_log_entry.user:
             raise StrikeTrackingError
 
         applied_action_user: discord.User | discord.Member = audit_log_entry.user
 
-        if applied_action_user == self.bot.user:
+        if applied_action_user == self.tex_bot.user:
             return
 
         fetch_log_channel_error: RuntimeError
         try:
             confirmation_message_channel: discord.DMChannel | discord.TextChannel = (
                 await self.get_confirmation_message_channel(applied_action_user)
                 if applied_action_user != strike_user
-                else await self.bot.fetch_log_channel()
+                else await self.tex_bot.fetch_log_channel()
             )
         except RuntimeError as fetch_log_channel_error:
             raise StrikeTrackingError(
                 str(fetch_log_channel_error),
             ) from fetch_log_channel_error
 
-        MODERATION_ACTIONS: Final[Mapping[discord.AuditLogAction, str]] = {
-            discord.AuditLogAction.member_update: "timed-out",
-            discord.AuditLogAction.auto_moderation_user_communication_disabled: "timed-out",
-            discord.AuditLogAction.kick: "kicked",
-            discord.AuditLogAction.ban: "banned",
-        }
-
         member_strikes: DiscordMemberStrikes = (  # type: ignore[assignment]
             await DiscordMemberStrikes.objects.aget_or_create(
                 discord_id=strike_user.id,
             )
         )[0]
 
-        strikes_out_of_sync_with_ban: bool = bool(
+        STRIKES_OUT_OF_SYNC_WITH_BAN: Final[bool] = bool(
             (action != discord.AuditLogAction.ban and member_strikes.strikes >= 3)
-            or (action == discord.AuditLogAction.ban and member_strikes.strikes > 3),
+            or (action == discord.AuditLogAction.ban and member_strikes.strikes > 3)  # noqa: COM812
```

Comment:
why are we removing the trailing commas here? (and elsewhere)

Reply:
I think I'd rather not have the comma but then having to have the ignore comment looks just as stupid so tbh I don't know which is worse

Reply:
I prefer the ignore, purely because it gets coloured as a darker gray in my editor so hides more into the background. It also makes it clear that the thing on the end there is purely about formatting because it is a comment & not actual code.